### DISCUSSION TITLE
"Remove from playlist"->"All" not working

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#373] [Fixed Issues]
**Audio 1 • 20210603
5.10.39-2-ARCH aarch64
Raspberry Pi 3 Model B Rev 1.2**

Walkaround: "Remove from playlist"->"Crop" + "Clear playlist"->"Clear"

**Comment by EarsofTin** [comment]
Observing same problem immediately after 20210603 update.  Using 64bit version on RPi3 with Chrome and Brave browsers.

**Comment by klassikmann** [comment]
**Audio 1 • 202106041
Fix - Clear Playlist**

Problem fixed (whereas 20210604 some minutes before generates syntax errors in install.sh).

**Comment by EarsofTin** [comment]
All is well, thanks for the fix
